doc/classes/LinkButton.xml

```diff
@@ -26,7 +34,16 @@
 			Base text writing direction.
 		</member>
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
-			Determines when to show the underline. See [enum UnderlineMode] for options.
+			The underline mode to use for the text. See [enum LinkButton.UnderlineMode] for the available modes.
+		</member>
+		<member name="url" type="String" setter="set_url" getter="get_url" default="&quot;&quot;">
+			The URL for this [LinkButton]. If a URL has been set, it will be opened automatically with the operating system's default program assigned to the URL protocol when the button is pressed. For HTTP and HTTPS URLs, this is typically the user's default Web browser. See [method OS.shell_open] for more information.
+			Examples
+			[codeblock]
+			set_url("http://godotengine.org")
+			set_url("C:\SomeFolder")
+			set_url("C:\someImage.png")
+			[/codeblock]
```

Comment:
```suggestion
			[codeblocks]
			[gdscript]
			# Opens the URL in a web browser.
			url = "http://godotengine.org"
			# Opens the file explorer.
			url = "C:\SomeFolder"
			# Opens an image viewing software.
			url = "C:\someImage.png"
			[/gdscript]
			[csharp]
			// Opens the URL in a web browser.
			Url = "http://godotengine.org";
			// Opens the file explorer.
			Url = "C:\SomeFolder";
			// Opens an image viewing software.
			Url = "C:\someImage.png";
			[/csharp]
			[/codeblocks]
```

Please encourage users to use properties. I also added descriptors for the desired effect. I am not so sure about the paths being Windows specific. Maybe we should use relative paths instead.
Personally, I am all in for more examples where people read them when they search for the desired functionality. Though this is essentially mimicking the shell_open documentation.